client/app/dbaas/logs/detail/streams/alerts/add/alerts-add.html

```diff
@@ -1,5 +1,195 @@
+                <div class="oui-field__content">
+                    <div class="oui-field-control oui-field-control_4">
+                        <div class="oui-select">
+                            <select class="oui-select__input"
+                                name="threshold_type"
+                                id="threshold_type"
+                                data-ng-model="ctrl.alert.thresholdType">
+                                <option data-ng-if="ctrl.alertType !== 'FIELD_VALUE'" value="MORE" data-translate="add_alert_threshold_type_more"></option>
+                                <option data-ng-if="ctrl.alertType !== 'FIELD_VALUE'" value="LESS" data-translate="add_alert_threshold_type_less"></option>
+                                <option data-ng-if="ctrl.alertType === 'FIELD_VALUE'" value="LOWER" data-translate="add_alert_threshold_type_lower"></option>
+                                <option data-ng-if="ctrl.alertType === 'FIELD_VALUE'" value="HIGHER" data-translate="add_alert_threshold_type_higher"></option>
+                            </select>
+                            <i class="oui-icon oui-icon-chevron-down" aria-hidden="true"></i>
+                        </div>
+                    </div>
+                    <div class="oui-field-helper" data-translate="{{ 'add_alert_threshold_type_field_description_' + ctrl.alertType.toLowerCase() }}"></div>
+                </div>
+            </div>
+
+            <div class="oui-field" data-ng-class="{'oui-field_error': ctrl.form.$submitted && ctrl.form.threshold.$invalid}">
+                <label for="threshold" class="oui-field__label oui-label" data-translate="add_alert_threshold_field"></label>
+                <div class="oui-field__content">
+                    <oui-numeric
+                        id="threshold"
+                        name="threshold"
+                        model="ctrl.alert.threshold"
+                        min="0"
+                        max="99"
+                        required>
+                    </oui-numeric>
+                    <div class="oui-field-helper" data-translate="add_alert_threshold_field_description"></div>
+                </div>
+            </div>
+
+            <div data-ng-if="ctrl.alertType === 'FIELD_VALUE'" class="oui-field">
+                <label for="constraint_type" class="oui-label" data-translate="add_alert_constraint_type_field"></label>
+                <div class="oui-field__content">
+                    <div class="oui-field-control oui-field-control_4">
+                        <div class="oui-select">
+                            <select class="oui-select__input"
+                                name="constraint_type"
+                                id="constraint_type"
+                                data-ng-model="ctrl.alert.constraintType">
+                                <option value="MEAN" data-translate="add_alert_constraint_type_mean"></option>
```

Comment:
This list of options should be in the controller.
If, one day, the Enum comes from the API, you will be able to change it more easily.

Reply:
This is done. The controller returns the options from the constants